sklearn/model_selection/tests/test_search.py

```diff
@@ -752,14 +752,14 @@ def check_cv_results_array_types(search, param_keys, score_keys):
     scorer_keys = search.scorer_.keys() if search.multimetric_ else ['score']
 
     for key in scorer_keys:
-        assert_true(cv_results['rank_test_%s' % key].dtype == np.int32)
+        assert cv_results['rank_test_%s' % key].dtype == np.int32
 
 
 def check_cv_results_keys(cv_results, param_keys, score_keys, n_cand):
     # Test the search.cv_results_ contains all the required results
     assert_array_equal(sorted(cv_results.keys()),
                        sorted(param_keys + score_keys + ('params',)))
-    assert_true(all(cv_results[key].shape == (n_cand,)
+    assert all(cv_results[key].shape == (n_cand,
```

Comment:
invalid syntax here I think?

Reply:
also I see that the assert change from the original, is not?

Reply:
yeah -- the 'cons' of auto sed. will need to fix them up - didn't spot on a brief eye balling